src/cmd/run.js

```diff
@@ -63,23 +68,48 @@ export function defaultWatcherCreator(
     sourceDir,
     artifactsDir,
     onChange: () => {
```

Comment:
It looks that this can now be rewritten as a one line:

```
...
onChange: () => addonReload({addonId, client}),
...
```

(when the arrow function is composed by a single expression, we can omit the curly brackets and the arrow function will return the value of that single expression without the need to explicitly return it).